hardhat.config.js

```diff
@@ -30,5 +30,12 @@ module.exports = {
       url: `https://rinkeby.infura.io/v3/${INFURA_API_KEY}`,
       accounts: testAccounts
     }
-  }
+  },
+  contractSizer: {
+  alphaSort: false,
+  disambiguatePaths: false,
+  runOnCompile: false,
+  strict: true,
+  only: [],
+}
```

Comment:
sort the formatting here. lines 35-39 should be indented

Reply:
Thanks, done.

Also, we have not followed the coding styles in many other files 😄. Nothing urgent for now but to keep in mind for the future. We're not consistenly adhering to what we decided earlier. In this file, I used space because it's what the rest of the config uses.

Reply:
there's other examples of nested parts of javascript files being on the same indentation level as their parents? because that's gross if true

Reply:
no, spaces instead of tabs ;) the most important question in the world of software engineering